src/main/scala/net/psforever/objects/equipment/EffectTarget.scala

```diff
@@ -65,7 +85,7 @@ object EffectTarget {
 
     private def CommonRepairConditions(v: Vehicle): Boolean = {
       v.Health > 0 && v.Health < v.MaxHealth &&
-        (v.History.findLast { entry => entry.isInstanceOf[DamagingActivity] } match {
+        (v.History.find { entry => entry.isInstanceOf[DamagingActivity] } match {
```

Comment:
Since history is organized EARLIEST -> LATEST, I'm concerned this is always only going to find the first damaging event, rather than comparing against the latest damaging event.

Reply:
It was just a "worth a shot" change that had the desired outcome. Each time I damaged the vehicle the repairing stopped. Before it would only delay repairs the first time it was damaged and if the repair attempt was started within 5 seconds of that. After 5 seconds it would keep repairing while taking damage.

Reply:
I see now.  I began switching around the order of events to be OLD + NEW instead of NEW + OLD, that is, normally appending actions to the history rather than prepending actions.  I didn't completely switch it around in all places before my latest major feature PR apparently.  In the update, however, I desire a OLD + NEW organization to history and exp evaluations were written with that in mind.

Reply:
I'll switch it back and know it'll work as expected again after yours is also eventually merged. I don't know the full effect of some of these things I change. I just try it, log in and see if I get what I was hoping for and hope it doesn't cause issues elsewhere 😂